### PR TITLE
Cockpit 197

### PR DESCRIPTION
**Pull request by KKoukiou**
Preview available here https://kkoukiou.github.io/cockpit-project.github.io/blog/cockpit-197.html

### CODE REVIEW

File: _posts/2019-06-26-cockpit-197.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+slug: cockpit-197
+category: release
+summary: Machines new features
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 197.
+
+### Machines: Support disks that reside on all pool types when adding disks to VMs
```
Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: This heading is long.  Something shorter like "support all storage pool types" or something is better, with the full explanation in the text below.

File: _posts/2019-06-26-cockpit-197.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+A cockpit user can now add a disk to a VM either by creating a new volume or
+by choosing an existing volume which can reside on [all supported by libvirt pool
+types](https://libvirt.org/storage.html#StorageBackendDir).
```
Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: "all supported by libvirt pool types" doesn't read very nicely.  maybe "any libvirt-supported pool type".

File: _posts/2019-06-26-cockpit-197.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+![Disk on NFS](/images/disk-on-nfs.png)
+
+![Disk on ISCSI](/images/disk-on-iscsi.png)
+
+### Machines: When creating a new VM there is indication about the available space
```
Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: Again, a bit too long, maybe "available space indicator for VM creation"

File: _posts/2019-06-26-cockpit-197.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+When choosing to create a new disk during VM creation the dialog has now
```
Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: When choosing to create → When creating

Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: has now → now has

File: _posts/2019-06-26-cockpit-197.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+indication regarding the available space on the destination storage.
```
Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: indication regarding → an indication of

File: _posts/2019-06-26-cockpit-197.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+This prevents the users from overallocating storage to VMs which will lead to
```
Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: will lead → would lead

File: _posts/2019-06-26-cockpit-197.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly. Here are the release notes from version 197.
+
+### Machines: Support all storage pool types for new disks
+
+A cockpit user can now add a disk to a VM either by creating a new volume or
+by choosing an existing volume which can reside on [any libvirt-supported pool
```
Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: Sorry.  Would go even a bit further: "by choosing an existing volume, which can now reside on"  (ie: add a comma and "now") to emphasize that this is the change.

File: _posts/2019-06-26-cockpit-197.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+type](https://libvirt.org/storage.html#StorageBackendDir).
+
+![Disk on NFS](/images/disk-on-nfs.png)
+
+![Disk on ISCSI](/images/disk-on-iscsi.png)
+
+### Machines: Available space indicator for VM creation
+
+When creating a new disk during VM creation the dialog now has an indication
```
Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: would put a comma after "creation,".

File: _posts/2019-06-26-cockpit-197.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+of the available space on the destination storage.
+This prevents the users from overallocating storage to VMs which would lead to
```
Comment (allisonkarlitskaya) [inline _posts/2019-06-26-cockpit-197.md]: s/users/user/

File: _posts/2019-06-26-cockpit-197.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+
+A cockpit user can now add a disk to a VM either by creating a new volume or
+by choosing an existing volume which can reside on [any libvirt-supported pool
+type](https://libvirt.org/storage.html#StorageBackendDir).
+
+![Disk on NFS](/images/disk-on-nfs.png)
+
+![Disk on ISCSI](/images/disk-on-iscsi.png)
+
+### Machines: Available space indicator for VM creation
```
Comment (martinpitt) [inline _posts/2019-06-26-cockpit-197.md]: Let's put a verb here: "Show available space on host at VM creation"

**Pull request merged**